### DISCUSSION TITLE
[Visual Studio 22] Copilot does not tab complete in certain cases

### DISCUSSION THREAD

**Discussion by JKamsker** [community/community#51894] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Pressing tab here
![grafik](https://user-images.githubusercontent.com/11245306/229635818-d7424699-be1c-4f45-91df-fe2c46887b8b.png)

will complete to that:

![grafik](https://user-images.githubusercontent.com/11245306/229635870-8e07b9d4-aa48-42d4-9115-332d87cf7657.png)

**Comment by Akash1134** [comment]
Hi @JKamsker , Which version of Copilot and Visual Studio 22 are you using? To get started, please check out [this article](https://docs.github.com/en/copilot/getting-started-with-github-copilot?tool=visualstudio) for Copilot and Visual Studio.
